### DISCUSSION TITLE
How to set the SSH connection between server and remote host

### DISCUSSION THREAD

**Discussion by oussama-amine** [semaphoreui/semaphore#1006] [Q&A]
Hello, I'm trying to use ansible-semaphore on my CentOS server to manage some distant Cisco switches with playbooks, but I couldn't find how to set the SSH key properly.

First, I have created a key with type "none" and used it for my repository on github (enverything is fine till now). Then, I have created another key for the inventory with type "ssh" but i'm not sure what to put in "private key" section. Since I've already set an SSH conncetion before to access my distant switch via ansible (which works perfectly), I have used this same key which I founded in .ssh/known_hosts file as the "private key". Now I'm getting this error when trying to run the template :
![1](https://user-images.githubusercontent.com/61756655/182235710-4b317ffc-31cc-4eef-9c74-24ecdc849b19.png)

Here is my repositoriy (with key type "none"):
![2](https://user-images.githubusercontent.com/61756655/182236013-a663f775-0562-491a-9fce-a8c8558d48f7.png)

Here how set my ssh key :
![3](https://user-images.githubusercontent.com/61756655/182236296-b221a325-faa3-4b25-8561-721be3fa0d0c.png)

Here is my inventory : 
![4](https://user-images.githubusercontent.com/61756655/182236668-c78c47f8-e782-4596-86c5-21973ad2ace9.png)

Here is my playbook :
![5](https://user-images.githubusercontent.com/61756655/182237238-b6552efb-204c-4e08-ae47-a08851715a5d.png)

Can anyone help with this please, I have been trying for days.. Thank you very much.

**Comment by cf-sewe** [comment]
You SSH private key looks incorrect (its a public key). Once you correct it , login should work. 
You may also remove `ansible_password` from the inventory.

A private key would look similar to that:

```
-----BEGIN RSA PRIVATE KEY-----
<PRIVATE_KEY_DATA>
-----END RSA PRIVATE KEY-----
```

To keep it simple, make sure that the private key is not encrypted with a passphrase.
